qsirecon/interfaces/dsi_studio.py

```diff
@@ -501,27 +533,84 @@ def _sanitized_connectivity_matrix(conmat, official_labels):
         official_labels : ndarray (M,)
             Array of official ROI labels. The matrix in conmat will be reordered to
             match the ROI labels in this array
+        official_label_names : ndarray of str or None
+            Array of ROI names corresponding to official_labels. If None, assumes numeric labels.
 
     Returns:
     --------
         connectivity_matrix : ndarray (M, M)
             The DSI Studio data reordered to match official_labels
     """
     m = loadmat(conmat)
+    connectivity = m["connectivity"]
     n_atlas_labels = len(official_labels)
-    # Column names are binary strings. Very confusing.
-    column_names = "".join([s.decode("UTF-8") for s in m["name"].squeeze().view("S1")]).split(
-        "\n"
-    )[:-1]
-    matfile_region_ids = np.array([int(name.split("_")[-1]) for name in column_names])
 
-    # Where does each column go? Make an index array
-    connectivity = m["connectivity"]
-    in_this_mask = np.isin(official_labels, matfile_region_ids)
-    truncated_labels = official_labels[in_this_mask]
-    assert np.all(truncated_labels == matfile_region_ids)
+    # Column names are binary strings
+    column_names = (
```

Comment:
Where are these column names coming from? If they're from the NIfTI, shouldn't they be integers?